src/rprblender/engine/viewport_engine.py

```diff
@@ -554,7 +554,22 @@ def sync_update(self, context, depsgraph):
         updates = []
         for obj_type in (bpy.types.Scene, bpy.types.World, bpy.types.Material, bpy.types.Object,
                          bpy.types.Collection, bpy.types.Light):
-            updates.extend(update for update in depsgraph.updates if isinstance(update.id, obj_type))
+            for update in depsgraph.updates:
+                if isinstance(update.id, obj_type):
+                    updates.append((update.id, update.is_updated_geometry, update.is_updated_transform))
+
+        # despgraph doesn't provide updates for ShaderNodeTexImage with activated Auto Refresh option
```

Comment:
Getting materials with ShaderNodeTexImage node and source == 'SEQUENCE' has to be optimized. It should be used only when we have Scene object in updates and frame was changed.

Reply:
Done in new commit.